package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "react-trainings",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --open --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js"
```

Comment:
Basically, definition of different webpack configuration files can be skipped in the package.json. You can create `webpack.config.js` file and merge one of production or development configs according to the passed webpack mode value.